Source/App/EbAppProcessCmd.c

```diff
@@ -826,8 +826,8 @@ static void ReadInputFrames(
                     /* if not a y4m file and input is read from stdin, 9 bytes were already read when checking
                        or the YUV4MPEG2 string in the stream, so copy those bytes over */
                     memcpy(ebInputPtr, config->y4m_buf, YUV4MPEG2_IND_SIZE);
-                    headerPtr->nFilledLen += YUV4MPEG2_IND_SIZE;
                     ebInputPtr += YUV4MPEG2_IND_SIZE;
+                    headerPtr->nFilledLen += YUV4MPEG2_IND_SIZE;
```

Comment:
Do we still need this PR?

Reply:
I'll withdraw it - Klockwork will flag the memcpy usage, but I'll defer that issue.

-mark